lib/loader/loader.js

```diff
@@ -3,15 +3,37 @@ var loader, define, requireModule, require, requirejs;
 (function(global) {
   'use strict';
 
+  var heimdall = global.heimdall;
+  var {
+    define: __define,
+    require: __require,
+    exports: __exports,
+    findModule: __findModule,
+    resolve: __resolve,
+    reify,
+    findDeps,
+    modules,
+    resolveRelative,
+    pendingQueueLength
+  } = heimdall.registerMonitor('loaderjs',
+  'define',
+  'require',
+  'reify',
+  'findDeps',
+  'modules',
+  'exports',
+  'resolve',
+  'resolveRelative',
+  'findModule',
+  'pendingQueueLength');
```

Comment:
this destructuring seems to break PhantomJS compat for the bower release of loader.js as used in ember-qunit and ember-mocha

/cc @rwjblue